src/webserver/Static.jl

```diff
@@ -222,6 +222,22 @@ function http_router_for(session::ServerSession)
     end
     HTTP.@register(router, "GET", "/notebookfile", serve_notebookfile)
 
+    serve_statefile = with_authentication(; 
+        required=security.require_secret_for_access || 
+        security.require_secret_for_open_links
+    ) do request::HTTP.Request
+        try
+            notebook = notebook_from_uri(request)
+            response = HTTP.Response(200, MsgPack.pack(Pluto.notebook_to_js(notebook)))
+            push!(response.headers, "Content-Type" => "application/octet-stream")
+            push!(response.headers, "Content-Disposition" => "inline; filename=\"$(without_pluto_file_extension(basename(notebook.path))).plutostate\"")
```

Comment:
Cool! Can you add a `Content-Disposition` header to our other endpoints as well?

Reply:
It's already set for `/notebookexport` and `/notebookfile` as well, do you want it set for everything else too?

Reply:
I see, thanks!